Products/ZenUI3/browser/javascript.py

```diff
@@ -251,13 +251,18 @@ def snippet(self):
         # 3. the timezone of the browser
         user = self.context.dmd.ZenUsers.getUserSettings()
         timezone = user.timezone
+        date_fmt = user.dateFormat
         snippet = """
-            Zenoss.env.COLLECTORS = %r;
-            Zenoss.env.priorities = %r;
-            Zenoss.env.productionStates = %r;
-            Zenoss.USER_TIMEZONE = "%s" || jstz.determine().name();
-        """ % ( collectors, priorities, productionStates, timezone )
-        return snippet
+            var z = Ext.ns('Zenoss'),
+                env = Ext.ns('Zenoss.env');
+
```

Comment:
After chatting about it, I think the global `Zenoss` object should be created here as it was before instead of using Ext namespaces.